Proposal-Template.md

```diff
@@ -2,20 +2,51 @@
 
 All sections of this template are required. You can add more sections if necessary.
 
-| WRITE_PROPOSAL_#_HERE (e.g. Proposal_1) | WRITE_TITLE_HERE |
+| Proposal 3 | Returning to Voting on Rewards and Reverting the DAO Allocation |
 | --- | ----------- |
-| Current Status | CHOOSE_ONE Proposal(Draft, Final Comments, Dropped), Community Call & Final Merge, Voting, Approved, Failed|
-| Author | AUTHORS_NAME |
-| Date Created | DATE_CREATED |
-| Forum Link | [FORUM_LINK](https://forum.volatility.com/) |
-| Simple Summary | 1-2_Sentences|
+| Current Status | Final Comments |
+| Author | joe.78 |
+| Date Created | 9/14/21 |
+| Forum Link | https://forum.volatility.com/t/proposal-3-returning-to-voting-on-rewards-and-reverting-the-dao-allocation/74 |
+| Simple Summary | This proposal will return the decision of rewards allocations to the voters and will also revert the DAO Allocation back to rewards.|
 
 ## Abstract
+Proposal 3 is a new proposal based on the draft proposal on rewards submitted by #newnamesamegai and the comments on it in the github.  The original draft proposal has been up for comment since August 29, 2021.
 
 ## Motivation
+The passing of Proposal 1 had multiple effects on rewards.  Firstly, weekly voting was removed, along with the reward categories of Uniswap LP (was receiving ~0%), rewards on rewards and governance (these categories caused considerable work on the VOL team). Secondly, reward allocations were set at 50% SLP, 25% Staked VOL, and 25% DAO allocation.  While the proposal passed easily, there were concerns voiced fairly soon after the vote. Some of these were:
+
+1. There was not enough time to comment on the proposal.
+2. The DAO already has a treasury, does it need more?
+3. Many people enjoyed voting to influence the rewards.
+4. The % for SLP should be higher, due to the risks of providing liquidity.
+
+Proposal 3 will address these concerns, along with the concerns that created the initial proposal that passed.
 
 ## Specification or Details
+Proposal 3 may consist of 6 separate votes, Part A - Part F.
+Part C - Part D will only be voted on if Part B passes.
+Part E - Part F will only be voted on if Part D passes.
+
+**Part A: Should we return DAO Allocation to Community Rewards?** YES or NO
+A YES vote on Part A will return the 25% of weekly rewards currently allocated to the DAO back to Community Rewards. A NO vote on Part A will leave the 25% of weekly rewards going toward the DAO.
```

Comment:
To be clear all of these funds are the DAOs. Perhaps we can rephrase for more clarity? Would something like the following work:

A YES vote on Part A means that none of the weekly community tokens will be set aside for later use, all will be allocated for use for rewards. A NO vote on Part A will leave 25% of the weekly community tokens being set aside for future use.

Reply:
I made this change, it is under commits? I guess it has to be merged